packages/cosmic-swingset/lib/block-manager.js

```diff
@@ -7,6 +7,7 @@ const DELIVER_INBOUND = 'DELIVER_INBOUND';
 const END_BLOCK = 'END_BLOCK';
 const COMMIT_BLOCK = 'COMMIT_BLOCK';
 const IBC_EVENT = 'IBC_EVENT';
+const PLEASE_PROVISION = 'PLEASE_PROVISION';
```

Comment:
maybe we should add `PROVISIONED` and use these "constants" in vats/bootstrap too? They'd have to be exported from their own file, of course, but it might highlight the dependency between the place where the vats are defined and the place where the host communicates with them.